<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxWindow</class>
 <widget class="QFrame" name="TxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="1">
    <widget class="QPushButton" name="clearWindow">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>383</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QScrollArea" name="txArea">
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="txLines">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>484</width>
        <height>205</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="linesLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearWindow</sender>
   <signal>clicked()</signal>
   <receiver>TxWindow</receiver>
   <slot>clearBuffers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clearBuffers()</slot>
 </slots>
</ui>
